src/NRules/NRules.Testing/ISelectedRuleVerification.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace NRules.Testing;
+
+public interface ISelectedRuleVerification
```

Comment:
@gritcsenko not sure if "selected" adds anything here, vs just naming IRuleVerification. What do you think?

Reply:
Renamed, thanks